thrift/from_wire_test.go

```diff
@@ -230,6 +230,27 @@ func TestValueFromWireSuccess(t *testing.T) {
 				"s": "foo",
 			},
 		},
+		{
+			// Enum with recognized value.
+			w: wire.NewValueI32(1),
+			spec: &compile.EnumSpec{
+				Name: "Op",
+				Items: []compile.EnumItem{{
+					Name:  "Add",
+					Value: 1,
+				}},
+			},
+			v: "Add",
```

Comment:
Also for later: You probably want to link the spec with an empty scope before you use it. This is not actionable right now because this isn't doable without some amount of boilerplate (see https://github.com/thriftrw/thriftrw-go/issues/135).